locksmithctl/daemon.go

```diff
@@ -352,5 +372,14 @@ func runDaemon() int {
 	close(stop)
 	wg.Wait()
 
+	if period != nil {
+		now := time.Now()
+		sleeptime := period.DurationToStart(now)
+		if sleeptime > 0 {
+			fmt.Printf("Waiting until %s to reboot.\n", now.Add(sleeptime).UTC().Format(time.UnixDate))
```

Comment:
maybe too pedantic, but since sleep is monotonic it would be more accurate to say "waiting for xxx days/mins/etc"